cmd/launcher/web/web.go

```diff
@@ -201,6 +204,9 @@ func NewLauncher(sublaunchers ...Sublauncher) launcher.SubLauncher {
 
 	fs := flag.NewFlagSet("web", flag.ContinueOnError)
 	fs.IntVar(&config.port, "port", 8080, "Localhost port for the server")
+	fs.DurationVar(&config.writeTimeout, "write-timeout", 15*time.Second, "Server write timeout (i.e. '10s', '2m')")
```

Comment:
Let's add the comments to the flags:
1. Explaining what is read/write timeouts exactly do (sometimes it could be confusing what's for read and what's for write).
2. Explaining better which time format is expected (as it's a time.Duration it can be 1h or 200ms). We can e.g. add (`as in time.Duration format`). Let's see if it can be sufficiently short.